docs/installation.html

```diff
@@ -37,18 +37,18 @@ <h3>Prerequisites</h3>
 <p>Before installing, make sure you have the appropriate dependencies installed. First, there is OCaml:
 
 <ul>
-    <li>ocaml &ge; 4.07.0 &amp; &lt; 4.10.0</li>
+    <li>ocaml &ge; 4.08.0 &amp; &lt; 4.11.0</li>
 </ul>
 
 <p>The recommended way to install Ocaml is with OPAM. Instructions on downloading OPAM, and using OPAM to download Ocaml can be found <a href="https://ocaml.org/docs/install.html#OPAM">here</a>.
 
 <p>Then, there are the following OCaml packages:
 
 <ul>
-    <li>core_kernel 0.12.4+</li>
+    <li>core_kernel 0.14.0</li>
     <li>bap 2.1.0</li>
```

Comment:
I thought that it should be 2.2.0, but when I do `opam show bap`, it tells me that I am on version `master`, with the next most recent tagged release being 2.1.0. Because of this, I wasn't sure if I should change it to 2.2.0 on the site and the opam file. What are your thoughts?